R/utils.R

```diff
@@ -165,7 +165,7 @@ check_compat_exp <- function(explist) {
   # Test equality of experiments in list
   if (length(explist) > 1) {
     equal <- vapply(seq_along(explist)[-1], function(i) {
-      all.equal(explist[[1]]$IDX, explist[[i]]$IDX)
+      isTRUE(all.equal(explist[[1]]$IDX, explist[[i]]$IDX))
```

Comment:
Hey Angela, I have a small request for a change. To get around the version 3.5 problem; could you replace `isTRUE` with `literalTRUE` and add the following function to the end of utils.R? (no roxygen comments needed)

```
literalTRUE <- function(x) is.logical(x) && length(x) == 1L && !is.na(x) && x
```

Reply:
I added that commit to the pull request

Reply:
Very nice, thanks!